rotkehlchen/accounting/structures.py

```diff
@@ -312,6 +312,46 @@ class HistoryEventSubType(SerializableEnumMixin):
     SPEND = 6
     RECEIVE = 7
 
+    def serialize_event_subtype(self) -> str:
+        """Serialize event subtype to a readable string
+        May raise:
+        - DeserializationError
+        """
+        # TODO: change this when the changes to history entries get merged in the left branch
+        if self.name == 'REWARD':
+            return 'get reward'
+        description = EVENTS_SUBTYPES_TO_STR.get(
+            HistoryEventSubType[self.name],
+            UNKNOWN_SUBTYPE,
```

Comment:
Why have you added this? Why use a special string for "unknown subtype". Just use `None`.

Reply:
Didn't use None because in develop `None` still means `get reward`

Reply:
yes but this is type to str. And we are talking about the TO part here